src/graphql/mutation/mfa/enrollSMSAuthenticator.graphql

```diff
@@ -0,0 +1,8 @@
+mutation enrollSMSAuthenticator($mfa_token: String!, $phone_number: String!) {
+	my {
+		enrollSMSAuthenticator( mfa_token: $mfa_token, phone_number: $phone_number) {
```

Comment:
We'll want to also request the `recovery_codes` field so that we can display the recovery code step once #2436 is merged. That can be a follow-up pr to this one.